internal/website/gatherexamples/gatherexamples.go

```diff
@@ -14,22 +14,24 @@
 
 // Command gatherexamples extracts examples in a Go module into a JSON-formatted
 // object. This is used as input for building the Go CDK Hugo website.
-// Examples must include a comment that starts with "// This example is used in"
+//
+// Examples must include a comment
+// "// PRAGMA(gocloud.dev): Package this example for gocloud.dev."
 // somewhere in the function body in order to be included in this tool's output.
 //
 // gatherexamples does some minimal rewriting of the example source code for
 // presentation:
 //
 //   - Any imports the example uses will be prepended to the code.
 //   - log.Fatal(err) -> return err
-//   - A comment line "// Variables set up elsewhere:" will remove any code up
-//     to the next blank line. This is intended for compiler-mandated setup
-//     like `ctx := context.Background()`.
-//   - A comment line "// Ignore unused variables in example:" will remove any
-//     code until the end of the function. This is intended for
+//   - A comment line "// PRAGMA(gocloud.dev): Skip until next blank line." will
+//     remove any code up to the next blank line. This is intended for
+//     compiler-mandated setup like `ctx := context.Background()`.
```

Comment:
I still don't understand why we insist on excluding ctx from the examples :(

Reply:
I think it was just to make the examples shorter. The assumption is that everyone knows how to get a `ctx` so it doesn't add any value to show that line every time.

For this and the `log.Fatal` question, we can reconsider -- file a separate issue though. I'm trying to keep this PR not changing the output at all.